api-server/migrations/20210828215333-refactor-date-storage.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.query(`
+      CREATE VIEW public.datetime_venue AS
+      SELECT public.uuid_generate_v4() as id,
+        id AS event_id,
+        venue_id,
+        start_time,
+        end_time,
+        optional_title
+      FROM
+          (SELECT * FROM events,
+            jsonb_to_recordset(events.date_times) AS dates(start_time timestamp, end_time timestamp, optional_title character varying(255)))
```

Comment:
that `jsonb_to_recordset` is super helpful. I can't believe I didn't know about that 👍